sdk/src/trace/span.cc

```diff
@@ -62,13 +62,15 @@ Span::Span(std::shared_ptr<Tracer> &&tracer,
            nostd::string_view name,
            const trace_api::KeyValueIterable &attributes,
            const trace_api::StartSpanOptions &options,
-           const trace_api::SpanContext &parent_span_context) noexcept
+           const trace_api::SpanContext &parent_span_context,
+           const nostd::span<std::shared_ptr<trace_api::Link>> &links) noexcept
     : tracer_{std::move(tracer)},
       processor_{processor},
       recordable_{processor_->MakeRecordable()},
       start_steady_time{options.start_steady_time},
       has_ended_{false}
 {
+  (void)options;
```

Comment:
Not sure why this is added back, should be removed because it was referenced in line 108 and 109.

Reply:
Yeah my bad . Thanks for pointing. Have raised a separate PR #351 as this was now a rework. Please review that. Will close this one.